testplan/exporters/testing/pdf/renderers/reports.py

```diff
@@ -111,31 +115,45 @@ def get_row_data(self, source, depth, row_idx):
             ],
         )
 
+        samplestyles = styles.getSampleStyleSheet()
+        metadata = [
+            [key, Paragraph(value, samplestyles["Normal"]), "", ""]
+            for key, value in self.get_metadata_context(source).items()
+        ]
+
         # Metadata
-        row_data.append(
-            content=[
-                [key, value, "", ""]
-                for key, value in self.get_metadata_context(source).items()
-            ],
-            style=[
-                RowStyle(
-                    bottom_padding=0,
-                    left_padding=0,
-                    top_padding=0,
-                    valign="TOP",
-                ),
-                RowStyle(
-                    font=(const.FONT_BOLD, const.FONT_SIZE_SMALL),
-                    start_column=0,
-                    end_column=0,
-                ),
-                RowStyle(
-                    font=(const.FONT, const.FONT_SIZE_SMALL),
-                    start_column=1,
-                    end_column=1,
-                ),
-            ],
-        )
+        for key, value in self.get_metadata_context(source).items():
+            row_data.append(
+                content=[
+                    key,
+                    Paragraph(value, samplestyles["Normal"]),
+                    "",
+                    "",
+                ],
+                style=[
+                    RowStyle(
+                        bottom_padding=0,
+                        left_padding=0,
+                        top_padding=0,
+                        valign="TOP",
+                    ),
+                    RowStyle(
+                        font=(const.FONT_BOLD, const.FONT_SIZE_SMALL),
+                        start_column=0,
+                        end_column=0,
+                    ),
+                    RowStyle(
+                        font=(const.FONT, const.FONT_SIZE_SMALL),
+                        start_column=1,
+                        end_column=1,
+                    ),
+                    RowStyle(
```

Comment:
Because of the behaviour of RowData.append() function the RowStyles are getting merged for columns if we pass a list of the rows as content. This is bot acceptable for span style because it would merge all columns. So we either add the columns separately or we manually add the span styles to the RowData instance after inserting the rows.